docs/api-reference/javascript/v2/interfaces/HLSConfig.md

```diff
@@ -24,5 +24,5 @@ pass in this field if recording needs to be turned on as well
 
 • `Optional` **variants**: [`HLSMeetingURLVariant`](/api-reference/javascript/v2/interfaces/HLSMeetingURLVariant)[]
 
-A list of meeting url which needs to be streamed as HLS feed, only one url is currently supported, all entries
```

Comment:
api-reference changes will get overriden next time the wf is run

Reply:
Got it, will update the code comments